frontend/screens/DogProfile.tsx

```diff
@@ -29,6 +31,17 @@ const DogProfile = ({ navigation }: Props) => {
   const [ownerName, setOwnerName] = useState("");
   const [buttonSelected, setSelect] = useState(false);
 
+  //useContext stuff
+  const { UserData, setUserData } = useUserData()!;
+
+  const handleChange = (text: string, name: string): void => {
```

Comment:
Can we declare this handleChange function in another file and then import it into each of our create screens? That way, it will reduce code duplication.

Reply:
I think I tried doing that by putting it in UserContext.tsx, but I couldn't get it working. Would you happen to have any ideas?

Reply:
I'll try that out tomorrow and let you know if it works.